src/languages/javascript.js

```diff
@@ -592,7 +592,7 @@ export default function(hljs) {
         relevance: 0
       },
       {
-        match: [ /\bconstructor(?=\s*\()/ ],
+        match: [ new RegExp(`\\b(constructor|${ECMAScript.BUILT_IN_GLOBALS.map(x=>`${x}\\w+`).join('|')}|${ECMAScript.BUILT_IN_VARIABLES.map(x=>`${x}\\w+`).join('|')})(?=\\s*\\()`) ],
```

Comment:
Why so limited though vs matching just any valid function name?

```
[IDENT]()
```

Reply:
This is against ```js
  const FUNCTION_CALL = {
    match: regex.concat(
      /\b/,
      noneOf([
        ...ECMAScript.BUILT_IN_GLOBALS,
        "super",
        "import"
      ]),
      IDENT_RE, regex.lookahead(/\(/)),
    className: "title.function",
    relevance: 0
  };
  ```

Reply:
Oh right, we already do this... so isn't this just a bug/problem with the `noneOf` guard?  This seems to fix it:

```js
noneOf([
        ...ECMAScript.BUILT_IN_GLOBALS,
        "super",
        "import"
      ].map(x => `${x}\\(`)),
```

Thoughts?

Reply:
More complete to deal with spaces also:

```
  const FUNCTION_CALL = {
    match: regex.concat(
      /\b/,
      noneOf([
        ...ECMAScript.BUILT_IN_GLOBALS,
        "super",
        "import"
      ].map(x=> `${x}\\s*\\(`)),
      IDENT_RE, regex.lookahead(/\s*\(/)),
    className: "title.function",
    relevance: 0
  };
```

Reply:
It looks good. I amended as what you suggested.